src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerSaveImage.groovy

```diff
@@ -20,10 +24,21 @@ class DockerSaveImage extends AbstractDockerRemoteApiTask {
      * The image including repository, image name and tag to be saved e.g. {@code vieux/apache:2.0}.
      *
      * @since 6.0.0
+     * @deprecated Use {@link #images} instead
      */
     @Input
+    @Optional
+    @Deprecated(forRemoval = true)
```

Comment:
The next release will likely be a major version upgrade. While it's usually better to deprecate first, I might just remove this field right away as a breaking change.

Reply:
Sure, no problem, makes it slightly easier of course.